AK/Utf8View.cpp

```diff
@@ -0,0 +1,78 @@
+#include <AK/Utf8View.h>
+
+Utf8View::Utf8View(StringView string)
+    : m_string(string)
+{
+}
+
+AK::Utf8CodepointIterator Utf8View::begin() const
```

Comment:
Let's wrap this whole file in `namespace AK {}` instead, then you don't have to write out `AK::` before things.